dsc-linux-5.4.173/patches-2023-01/01_elba_kpcimgr.patch

```diff
@@ -0,0 +1,1463 @@
+From 19bc58d0416d79ad3dc66d5be3bb641be86576a5 Mon Sep 17 00:00:00 2001
+From: Hiren Mehta <hiren@pensando.io>
+Date: Fri, 13 Jan 2023 16:41:44 -0800
+Subject: [PATCH] drivers/soc/pensando: kpciemgr driver.
+
+kpciemgr driver enables the relocation of the module code to handle
+Elba indirect PCIe transactions.
```

Comment:
It would be more consistent to call this the `kpcimgr` driver in the subject/description.  (I would have called it kpciemgr too, but Oracle went with kpcimgr and that is what we have now.)

Reply:
fixed